ui/tests/acceptance/enterprise-namespaces-test.js

```diff
@@ -71,143 +68,6 @@ module('Acceptance | Enterprise | namespaces', function (hooks) {
     await deleteNSFromPaths(namespaces);
   });
 
-  test('it filters namespaces based on search input', async function (assert) {
```

Comment:
This is a pretty big test are we confident in removing it? I'm guessing it's mostly just duplicating coverage for the search input component?

Reply:
yep, lots of duplication in the component test. I double checked with Shannon.